### PR TITLE
Jnci ruby edits 10182018

### PR DESCRIPTION
**Pull request by jamesneimanconsulting**
@optimizely-christine Kindly review these changes and merge if approved. Please ask Mike Ng to review.
Please see https://optimizely.atlassian.net/projects/EDU/issues/EDU-2372 for more information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,266 @@
...
+
+* The loaded application initializes the Optimizely manager by retrieving the datafile.
+* The `index` page populates and displays the catalog item list.
+* When applying a filter, data is posted to the `shop` page and sorts the catalog items through `optimizely.activate()`. The application then navigates to the `index` page.
+* Clicking on a **Buy Now** button tracks the purchase using `optimizely.track()` and sends a conversion event for the event named `purchased_item`. The application then navigates to the **buy** page.
+
+## Prerequisites
+
+* Ruby with Gems
+* Github account configured with [SSH keys](https://help.github.com/articles/connecting-to-github-with-ssh/)
```
Comment (aliabbasrizvi) [inline README.md]: nit. GitHub.

File: README.md [hunk 2]
```diff
@@ -1,25 +1,266 @@
...
+4. Start the app server:
+```shell
+ruby bin/app.rb 
+```
+
+5. Navigate to [http://localhost:8080](http://localhost:8080) in your browser.
+
+**Note:**
+
+- You can update the Optimizely variation code in the [bin/app.rb](bin/app.rb) file.
```
Comment (aliabbasrizvi) [inline README.md]: May be show where variation code lives by mentioning code block here.

**Review by aliabbasrizvi — CHANGES_REQUESTED** [review]
The other demo app descriptions talk a lot about set up. This one does not go into details about set up on Optimizely. Thoughts @jamesneimanconsulting ?

**Review by aliabbasrizvi — CHANGES_REQUESTED** [review]
Setting to revise so that you can take a look.

**Comment by jamesneimanconsulting** [comment]
@aliabbasrizvi I have pushed another commit to this PR. Kindly let me know if there are other changes required.

**Pull request closed without merging**